doc/building.md

```diff
@@ -67,12 +67,12 @@ To update the version of a given package, use the following snippet
 
 where:
 - `$PackageName` is the name of the package, e.g. Microsoft.UI.Xaml
-- `$OldVersionNumber` is the version number currently used, e.g. 2.5.0-prerelease.200609001
-- `$NewVersionNumber` is the version number you want to migrate to, e.g. 2.4.200117003-prerelease
+- `$OldVersionNumber` is the version number currently used, e.g. 2.4.0-prerelease.200506002
+- `$NewVersionNumber` is the version number you want to migrate to, e.g. 2.5.0-prerelease.200812002
 
 Example usage:
 
-`git grep -z -l Microsoft.UI.Xaml | xargs -0 sed -i -e 's/2.5.0-prerelease.200609001/2.4.200117003-prerelease/g'`
+`git grep -z -l Microsoft.UI.Xaml | xargs -0 sed -i -e 's/2.4.0-prerelease.200506002/2.5.0-prerelease.200812002/g'`

```

Comment:
Those versions don't refer to actual packages because the first two digits are the year, the second two digits are month, and the next two digits are the day. By changing the last digit, we are actually referencing the second package of that day which will not exist, thus wouldn't be a valid upgrade/downgrade path. This avoids people accidently changing the values while updating the WinUI version.